frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -623,11 +705,19 @@ function DataFrame({
             clearTooltip()
           }}
           // Header click is used for column sorting:
-          onHeaderClicked={
-            // Deactivate sorting for empty state and for large dataframes:
-            isEmptyTable || isLargeTable ? undefined : sortColumn
-          }
+          onHeaderClicked={(colIndex: number, _event) => {
```

Comment:
is it possible to type the `_event` argument, e.g. `MouseEvent` or something like that?

Reply:
Nope, I don't think its tracked currently. I can add that after the release